test/OpenTelemetry.ResourceDetectors.Azure.Tests/OpenTelemetry.ResourceDetectors.Azure.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net7.0;net6.0;</TargetFrameworks>
+    <IncludeSharedTestSource>true</IncludeSharedTestSource>
```

Comment:
Just for my understanding, why the property `IncludeSharedTestSource` is used?

Reply:
copy paste :). Removed it.